modules/microshift-greenboot-check-status.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/checking-greenboot-scripts-status.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-check-status_{context}"]
+= Checking the status of Greenboot health checks
+
+Check the status of Greenboot health checks before making changes to the system or during troubleshooting. You can use any of the following several commands to help you ensure that Greenboot scripts have finished running.
+
+.Procedure
+
+* To see a report of health check status, use the following command:
++
+[source,terminal]
+----
+$ systemctl show --property=SubState --value greenboot-healthcheck.service
+----
++
+** An output of "start" means that Greenboot checks are still running.
+** An output of "exited" means that checks have passed and Greenboot has exited. Greenboot runs the scripts in `green.d` when the system is a healthy state.
+** An output of "failed" means that checks have not passed. Greenboot runs the scripts in `red.d` when the system is in this state and might restart the system.
```

Comment:
```suggestion
** An output of `failed` means that checks have not passed. Greenboot runs the scripts in `red.d` when the system is in this state and might restart the system.
```

Reply:
I think those would all fall under [literal values](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference:~:text=Inline%20commands%2C%20operations%2C-,literal%20values,-%2C%20variables%2C%20parameters%2C%20settings)

Reply:
Yeah, I suppose those qualify as system items. Good catches, thank you.